CONTRIBUTING.md

```diff
@@ -35,43 +35,4 @@ Before We Can Merge
 
 If you've had a pull request reviewed and accepted, congratulations! Before we can merge your changes, we'll need you to rebase off `origin/develop` and squash your commits into one. This will give us a cleaner git history.
```

Comment:
Potentially change to "rebase onto" to match how it's referred to elsewhere (like in the universe)